docs/questions/query-builder/expressions/now.md

```diff
@@ -0,0 +1,109 @@
+---
+title: Now
+---
+
+# Now
+
+`now` returns the current datetime using your Metabase [report timezone](../../../configuring-metabase/localization.md#report-timezone).
+
+## Creating conditional logic using the current date or time
+
+Let's say you have some project data, and you want to add a status column for each task. We'll assume today's date and time is November 22, 2022, 12:00:00.
+
+| Task     | Start                       | Deadline                    | Status          |
+|----------|-----------------------------|-----------------------------|-----------------|
+| Draft    | November 1, 2022, 12:00:00  | November 30, 2022, 12:00:00 | In progress     |
+| Review   | November 15, 2022, 12:00:00 | November 19, 2022, 12:00:00 | Needs extension |
+| Edit     | November 22, 2022, 12:00:00 | November 22, 2022, 12:00:00 | DUE RIGHT NOW!  |
+
+To mark a task in progress, you'd use the expression:
+
+```
+now >= [Start] AND now < [Deadline] 
+```
+
+To check if you need to ask for an extension:
+
+```
+now >= [Start] AND now >= [Deadline]
+```
+
+If you're looking for an adrenaline rush (and you have real-time data), you can flag the tasks that are due _right this second_:
+
+```
+now = [Deadline]
+```
+
+To set up the **Status** column that combines all three situations above, you'd wrap everything in a `case` expression:
+
+```
+case(now >= [Start] AND now < [Deadline], "In progress",
+     now >= [Start] AND now >= [Deadline], "Needs extension",
+     now = [Deadline], "DUE RIGHT NOW!")
+```
+
+## Accepted data types
+
+| [Data type](https://www.metabase.com/learn/databases/data-types-overview#examples-of-data-types) | Works with `now`  |
+| ----------------------- | -------------------- |
+| String                  | ❌                   |
+| Number                  | ❌                   |
+| Timestamp               | ✅                   |
+| Boolean                 | ❌                   |
+| JSON                    | ❌                   |
+
+This table uses `timestamp` and `datetime` interchangeably. If your dates and times are stored as strings or numbers in your database, you can [cast them to datetimes](../../../data-modeling/metadata-editing.md#casting-to-a-specific-data-type) from the Data Model page.
+
+## Limitations
+
+`now` might not actually be _now_ (in your local time) if you don't live in the same timezone as your Metabase [report time zone](../../../configuring-metabase/localization.md#report-timezone).
+
+If you need to compare `now` to a column in a different time zone, use [convertTimezone](./converttimezone.md) to shift both columns into the same time zone. For example:
+
+```
+convertTimezone(now, 'UTC', <report timezone>) >= convertTimezone([Deadline], 'UTC', <source time zone>)
+```
+
+`now` is only available on Mongo versions 4.2 or greater.
```

Comment:
```suggestion
### Note on `now` in MongoDB

`now` is only available on Mongo versions 4.2 or greater.
```

Reply:
We don't support Mongo versions <4.2 anymore, so we can remove this note. https://github.com/metabase/metabase/blob/3e636378c539cbeec5b4d3a47e0d7f9605207d7f/docs/databases/connections/mongodb.md#L9